internal/datatransfer/monitor.go

```diff
@@ -68,6 +69,10 @@ func (m *Monitor) GetLastSuccessfulSyncTime(workloadName string) *time.Time {
 	return nil
 }
 
+func (m *Monitor) WorkloadStarted(workloadName string, report []*podman.PodReport) {
```

Comment:
Can you explain why this no-op function is necessary? Why the parameters are needed?